playbooks/files/get_key.py

```diff
@@ -0,0 +1,45 @@
+import os
+from argparse import ArgumentParser
+
+from boto3.session import Session
+from botocore.exceptions import ClientError
+from Crypto.PublicKey import RSA
+
+api_endpoint = 'https://obs.eu-de.otc.t-systems.com'
+bucket = 'obs-csm'
+
+
+def parse_params():
+    parser = ArgumentParser('OpenTelekomCloud S3 key downloader')
+    parser.add_argument('--key', '-k', default=None)
+    parser.add_argument('--output', '-o', default=None)
+    args = parser.parse_args()
+    return args
+
+
+def generate_private_key():
+    key = RSA.generate(2048)
```

Comment:
old version, need to use from master in customer-service-monitoring

Reply:
Done